gaslift/GASLIFT-03.DATA

```diff
@@ -0,0 +1,1116 @@
+TABDIMS                                                                         
+         1       1       150     60      1       60     1*     1               /
+-- ---------------------------------------------------------------------------------------------------------------------------------
+-- GROUP, WELL AND VFP TABLE DIMENSIONS                                                
+-- ---------------------------------------------------------------------------------------------------------------------------------
+--                                                                                
+--       WELL    WELL    GRUPS   GRUPS                                          
+--       MXWELS  MXCONS  MXGRPS  MXGRPW                                         
+WELLDIMS                                                                        
+         12      2       4       4                                             /                                                                                
+--
+--       PRODUCING VFP TABLES
+--       VFP     VFP     VFP     VFP     VFP     VFP
+--       MXMFLO  MXMTHP  MXMWFR  MXMGFR  MXMALQ  MXVFPTAB
+VFPPDIMS
+         6       3       3       1       5       1                             /
+-- ---------------------------------------------------------------------------------------------------------------------------------
+-- NUMERICAL AND RUN CONTROL OPTIONS                                                    
+-- ---------------------------------------------------------------------------------------------------------------------------------
+--
+--       SET STACK SIZE FOR LINEAR SOLVER
+--
+NSTACK                                                                          
+         4                                                                     /
+--
+--       DEFINE THE NUMBER OF ITERATIONS TO UPDATE WELL FLOW TARGETS
+--
+NUPCOL                                                                          
+         3                                                                     /
+-- ---------------------------------------------------------------------------------------------------------------------------------
+-- INPUT AND OUTPUT OPTIONS                                                   
+-- ---------------------------------------------------------------------------------------------------------------------------------
+--
+--       METRIC SYSTEM OF UNITS FOR BOTH INPUT AND OUTPUT 
+--
+METRIC
+--
+--       SWITCH ON THE UNIFIED INPUT FILES OPTION
+--
+UNIFIN                                                                          
+--
+--       SWITCH ON THE UNIFIED OUTPUT FILES OPTION
+--
+UNIFOUT                                                                          
+--
+--       PATH       PATH
+--       ALIAS      DIRECTORY FILENAME
+PATHS
+        'MODEL05'   '..\model5\include'                                        /

```

Comment:
Please create a new folder named include here, and copy over the include files from the model5 folder.

Reply:
I'll like to keep this for the time being, as I may use different models in generating the tests. We already have a couple of gas lifts in model5 and I was considering moving these any any others gas lifts tests under the same directory.

Reply:
Then I believe you should move them over in a not to soon future. People working in one folder may change include files at their will without knowing that they are in use in other places in the repo. I am open to being wrong about this, but then we should have a broader discussion where the other active contributors voice in. My preference would be that you copy the include files over now, and then just delete them again if you move the tests over to the model5 folder. In any case, I have made my point, and will respect your wish at this point.